code/game/turfs/simulated/floor.dm

```diff
@@ -30,10 +30,14 @@ var/list/icons_to_ignore_at_floor_init = list("damaged1","damaged2","damaged3","
 	var/broken = 0
 	var/burnt = 0
 	var/floor_tile = null //tile that this floor drops
-	var/list/broken_states = list("damaged1", "damaged2", "damaged3", "damaged4", "damaged5")
-	var/list/burnt_states = list()
+	var/list/broken_states
+	var/list/burnt_states
 
 /turf/open/floor/New()
+	if (!broken_states)
+		broken_states = list("damaged1", "damaged2", "damaged3", "damaged4", "damaged5")
+	if (!burnt_states)
+		burnt_states = list()
```

Comment:
this isn't a lazy init, this is me not wanting to init it if a child hasn't already init'ed it.

while I understand that mechanically they are the same, the reasoning isn't so i won't use that.